packages/plugin-workspace-tools/sources/commands/foreach.ts

```diff
@@ -231,11 +234,11 @@ export default class WorkspacesForeachCommand extends BaseCommand {
           stdout.end();
           stderr.end();
 
-          const emptyStdout = await stdoutEnd;
-          const emptyStderr = await stderrEnd;
+          await stdoutEnd;
+          await stderrEnd;
 
-          if (this.verbose && emptyStdout && emptyStderr)
-            report.reportInfo(null, `${prefix} Process exited without output (exit code ${exitCode})`);
+          if (this.verbose)
+            report.reportInfo(null, `${prefix} Process exited (exit code ${exitCode})`);
```

Comment:
If we do this, we can take the opportunity to print the time the process took if `enableTimers` is set

Reply:
good idea, i added the process duration, not sure about the log message, let me know if you have a better idea ?